Specs/Scene/parseBoundingVolumeSemanticsSpec.js

```diff
@@ -0,0 +1,208 @@
+      tile: {
+        boundingVolume: undefined,
+        minimumHeight: undefined,
+        maximumHeight: undefined,
+      },
+      content: {
+        boundingVolume: undefined,
+        minimumHeight: undefined,
+        maximumHeight: undefined,
+      },
+    });
+  });
+
+  it("parses minimum and maximum height", function () {
+    var tileClass = new MetadataClass({
+      id: "tile",
+      class: {
+        properties: {
+          tileMinimumHeight: {
+            type: "FLOAT32",
+            semantic: "TILE_MINIMUM_HEIGHT",
+          },
+          tileMaximumHeight: {
+            type: "FLOAT32",
+            semantic: "TILE_MAXIMUM_HEIGHT",
+          },
+          contentMinimumHeight: {
+            type: "FLOAT32",
+            semantic: "CONTENT_MINIMUM_HEIGHT",
+          },
+          contentMaximumHeight: {
+            type: "FLOAT32",
+            semantic: "CONTENT_MAXIMUM_HEIGHT",
+          },
+        },
+      },
+    });
+
+    var tileMetadata = new TileMetadata({
+      class: tileClass,
+      tile: {
+        properties: {
+          tileMinimumHeight: minimumHeight,
+          tileMaximumHeight: maximumHeight,
+          contentMinimumHeight: minimumHeight,
+          contentMaximumHeight: maximumHeight,
+        },
+      },
+    });
+
+    expect(parseBoundingVolumeSemantics(tileMetadata)).toEqual({
+      tile: {
+        boundingVolume: undefined,
+        minimumHeight: minimumHeight,
+        maximumHeight: maximumHeight,
+      },
+      content: {
+        boundingVolume: undefined,
+        minimumHeight: minimumHeight,
+        maximumHeight: maximumHeight,
+      },
+    });
+  });
+
+  it("parses bounding volumes", function () {
+    var tileClass = new MetadataClass({
+      id: "tile",
+      class: {
+        properties: {
+          tileBoundingBox: {
+            type: "ARRAY",
+            componentType: "FLOAT64",
+            componentCount: 12,
+            semantic: "TILE_BOUNDING_BOX",
+          },
+          contentBoundingSphere: {
+            type: "ARRAY",
+            componentType: "FLOAT64",
+            componentCount: 4,
+            semantic: "CONTENT_BOUNDING_SPHERE",
+          },
+        },
+      },
+    });
+
+    var tileMetadata = new TileMetadata({
+      class: tileClass,
+      tile: {
+        properties: {
+          tileBoundingBox: boundingBox,
+          contentBoundingSphere: boundingSphere,
+        },
+      },
+    });
+
+    expect(parseBoundingVolumeSemantics(tileMetadata)).toEqual({
+      tile: {
+        boundingVolume: {
+          box: boundingBox,
+        },
+        minimumHeight: undefined,
+        maximumHeight: undefined,
+      },
+      content: {
+        boundingVolume: {
+          sphere: boundingSphere,
+        },
+        minimumHeight: undefined,
+        maximumHeight: undefined,
+      },
+    });
+  });
+
+  it("only the first valid bounding volume is returned", function () {
```

Comment:
Is this the correct name for this test? To me, it seems like you're checking that bounding volumes are being assigned based on the order of precedence. (`box` -> `region` -> `sphere`)